### PR TITLE
New class IncompleteEnvelopeInfo added

### PR DESCRIPTION
**Pull request by btensay**
A new class called IncompleteEnvelopeInfo is added so that to create the correct objects list in the blob report of DTSBPS-392.

Existing test are updated as well.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/StaleBlobFinder.java [hunk 1]
```diff
@@ -43,6 +44,7 @@ public StaleBlobFinder(BlobServiceClient storageClient, ServiceConfiguration ser
             .map(blob -> new BlobInfo(
                     containerName,
                     blob.getName(),
+                    UUID.randomUUID().toString(),
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/StaleBlobFinder.java:47]: you not need any change in this class

File: src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/StaleBlobControllerTest.java [hunk 2]
```diff
@@ -107,6 +130,6 @@ void should_return_400_for_invalid_time() throws Exception {
             .andDo(print())
             .andExpect(status().isBadRequest());
         verifyNoMoreInteractions(staleBlobFinder);
-    }
 
+    }
```
Comment (mdayican) [inline src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/StaleBlobControllerTest.java:134]: formatting?

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/StaleBlobFinder.java (hunk 1 above)
Comment (btensay) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/StaleBlobFinder.java:47]: ok I will remove it.

File: src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/StaleBlobControllerTest.java (hunk 2 above)
Comment (btensay) [reply src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/StaleBlobControllerTest.java:134]: Taken care of.

**Comment by btensay** [comment]
Suggested changes are made and new pull request that includes changes made in this PR will be created.

**Pull request closed without merging**